.github/workflows/python-package.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python package
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        pytest
```

Comment:
We don't have a test suite (yet). IMO it would make more sense to add proper tests first in a separate PR and then make it part of the CI workflow.

Reply:
If there is no test then pytest fails. Disabling it now means wee need to remember to enable it later again. Doing as I proposed does not harm and prohibits that we forget to enable it.

For the test suite we need to decide how to do it. I tegration testing with mocked Airbox or plane unit testing.

Reply:
Ok, we can keep this in. Looks like I have access to a stove again so that makes it easier for me to help set up some tests soon.

Reply:
BTW: Don't know you knew, before discovering you lib, I started one myself. Check out the test I had there: https://github.com/lordyavin/hwam-smart-control/tree/main/tests

Testing against real hardware is nothing we can do in GitHub actions, so I tried to create some coverage by mocking the responses.